voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/VoiceMessageFactory.java

```diff
@@ -113,7 +113,9 @@ private VoiceMessageFactory() {
     }
 
     try {
-      return (VoiceOutput<?>) OUTPUT_MAP.get(type).newInstance();
+      VoiceOutput<?> voiceOutput = (VoiceOutput<?>) OUTPUT_MAP.get(type).newInstance();
+      voiceOutput.setMessage(document);
+      return voiceOutput;
```

Comment:
Shame on @davidwphillips for merging this pull request without actually looking at what was being merged.